### PR TITLE
new records

### PR DESCRIPTION
**Pull request by thea-m**
https://github.com/BetaMasaheft/Documentation/issues/2527

### CODE REVIEW

File: new/LIT6981PPSorcerers.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            <change who="DR" when="2024-05-01">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition">
+                <note>The following incipit and explicit are taken from <ref type="mss" corresp="BAVet128"/>.</note>
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <ab>
+                     በስመ፡ አብ፡ <gap reason="ellipsis"/>ጸሎት፡ ወስእለት፡ በእንተ፡ እለ፡ ይቀትሉ፡ ሰብአ፡ ወመሠርያን፡ <sic>ወ</sic>ዝውእቱ፡ ወዘያስተኃፍሮሙ፡ ላዕለ፡ መሠርያን፡ አስማተ፡ እግዚአብሔር።
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6981PPSorcerers.xml:59]: Why did you highlight the ወ before ዝውእቱ, but not the ወ before ዘያስተኃፍሮሙ. The second seems to me more odd than the first one.

File: new/LIT6981PPSorcerers.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6981PPSorcerers.xml:26]: Add witness in source desc.

File: new/LIT6982PPDiseases.xml [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6982PPDiseases" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against diseases and wounds</title>
+                <title xml:lang="gez" xml:id="t2">ጸሎት፡ ወስእለት፡ በእንተ፡ ድኅነት፡ እምሕማመ፡</title>
+                <title xml:lang="gez" corresp="#t2" type="normalized">Ṣalot wa-sǝʾlat baʾǝnta dǝḫnat ʾǝm-ḥǝmāma…</title>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6982PPDiseases.xml:10]: delete  '...'

File: new/LIT6983PPAsmat.xml [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6983PPAsmat.xml:26]: Add witness in source desc.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/LIT6981PPSorcerers.xml (hunk 1 above)
Comment (thea-m) [reply new/LIT6981PPSorcerers.xml:59]: I followed the catalogue's annotations

**Pull request merged**